app/views/matchings/show.html.erb

```diff
@@ -23,12 +22,12 @@
 						<td>
 							<div class="btn-group">
 								<button type="button" class="btn dropdown-toggle" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
-									<%= Time.parse(event["timestamp"]).strftime("%A, %B %d %Y %I:%M %p") %>
+									<%= Time.parse(event["timestamp"]).strftime("%A, %B %d %I:%M %p") %>
 								</button>
 								<div class="dropdown-menu">
 									<% @times.where(is_date: false).each do |time| %>
 										<a style="text-decoration: none">
-											<%= link_to time.date_time.strftime("%A, %B %d %Y %I:%M %p"), modify_project_matching_path(@project, [[index], [time.date_time.strftime("%A, %B %d %Y %I:%M %p")]]), method: :modify%>
+											<%= link_to time.date_time.strftime("%A, %B %d %I:%M %p"), modify_project_matching_path(@project, [[index], [time.date_time.strftime("%A, %B %d %Y %I:%M %p")]]), method: :modify%>
```

Comment:
Clicking on the modify matching link no longer works for, it tells me "route not found". This is because in `config/routes.rb` the route verb is "post" but here it's "modify" (which isn't a REST verb in the first place). This should be changed to `method: :post` (ideally put should be used for both, but put doesn't seem to have a nice way of using here, so wouldn't worry about using it)

Reply:
I was playing around with it, I think the error was because of the way I was loading in js libraries (worked with method: modify and properly loaded js), but ya agree with you changed it to method:post